.ddev/config.yaml

```diff
@@ -1,33 +1,31 @@
 name: tea
 type: php
 docroot: ""
-php_version: "8.1"
+php_version: "8.3"
 webserver_type: nginx-fpm
 router_http_port: "8081"
 router_https_port: "4434"
 xdebug_enabled: false
 additional_hostnames: []
 additional_fqdns: []
 database:
-  type: mariadb
-  version: "10.5"
-nfs_mount_enabled: false
-mutagen_enabled: false
+    type: mariadb
+    version: "10.5"
```

Comment:
I'd prefer to have the formatting cleanup in a separate PR as a pre-change in order to separate the formatting changes from the real configuration changes.

Reply:
Those changes comes from ddev upgrade, it automatically reformat this file